api/comments/views.py

```diff
@@ -79,6 +79,7 @@ class CommentDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, Comm
         is_abuse       boolean            has this comment been reported by the current user?
         has_children   boolean            does this comment have replies?
         can_edit       boolean            can the current user edit this comment?
+        new_mentions   list               guids of mentioned users in content of comment
```

Comment:
The documentation needs to be updated to reflect that `new_mentions` is a list of guids of all of the mentioned users that are present when the comment is saved and `old_mentions` is a list of guids of anyone who has already been mentioned in this comment. Having `old_mentions` ensures that a user will not be notified more than once for each comment. For example, even if a mention is removed in an edit and then added again, the user would only be notified of the first occurrence of the mention.